src/main/java/org/sagebionetworks/bridge/udd/helper/SesHelper.java

```diff
@@ -45,6 +59,20 @@ public final void setSesClient(AmazonSimpleEmailServiceClient sesClient) {
     }
 
     /**
+     * Sends a notice to the given account that no data could be found. This notice also gives basic instructions on
+     * how they could enable data.
+     *
+     * @param studyInfo
+     *         study info, used to construct the email message, must be non-null
+     * @param accountInfo
+     *         account to send the notice to, must be non-null
+     */
+    public void sendNoDataMessageToAccount(StudyInfo studyInfo, AccountInfo accountInfo) {
+        Body body = new Body().withHtml(new Content(NO_DATA_BODY_HTML)).withText(new Content(NO_DATA_BODY_TEXT));
+        sendEmailToAccount(studyInfo, accountInfo, body);
```

Comment:
Email sending code may someday make for a good shared library, if we can factor it out appropriately.